cmd/stackdriver-prometheus-sidecar/main.go

```diff
@@ -238,21 +238,24 @@ func main() {
 
 	_, err := a.Parse(os.Args[1:])
 	if err != nil {
-		fmt.Fprintln(os.Stderr, errors.Wrapf(err, "Error parsing commandline arguments"))
+		logLevel := promlog.AllowedLevel{}
+		logLevel.Set("error")
+		tmpLogger := promlog.New(logLevel)
+		level.Error(tmpLogger).Log("msg", "Error parsing commandline arguments", "err", err)
```

Comment:
This is contrived. What is the benefit?
Unless I'm missing something, I'd be inclined to leave this untouched. The rest of the changes are fine.

Reply:
Intention is to make log format consistent. But I agree it is contrived just for cases when flag parsing fails.